### ISSUE TITLE
Stored properties cannot be marked potentially unavailable with '@available'

### ISSUE THREAD

**Issue by Ritesh175862** [Fitpassu/react-native-stripe-payments#44]
Hi Team,
I am getting this error in MacBook pro m2 chip

Stored properties cannot be marked potentially unavailable with '@available'
<img width="911" alt="Screenshot 2023-01-09 at 3 18 44 PM" src="https://user-images.githubusercontent.com/51701629/211280795-6d9851c0-8fb6-480b-a5f9-08ea9cbc6687.png">

**Comment by viktorasl** [comment]
Hey @Ritesh175862, looks like the issue is not related to this library.
Swift does not allow to have stored property which is potentially unavailable.
Possible solution can be something (if you don't need ref of exact `CardScanningView` type) like:
```swift
lazy var cardScanningView: UIView? = {
   if #available(iOS 13.0, *) {
     // place the implementation here
   } else {
     return nil
   }
}()
```

**Comment by viktorasl** [comment]
Sorry, that was related to Stripe library native code.
I've released 1.1.0 lib version with Stripe 20.x iOS and Android support. Although I recommend migrating to official Stripe RN library.
